### PR TITLE
Adding option to send number of retries to override default value of 3. 

### PR DESCRIPTION
**Pull request by chowmean**
This PR addresses https://github.com/kubernetes-client/python/issues/652
Adding option in configuration and rest client object to send number of retries value to override the default value of 3 in urllib3.

### CODE REVIEW

**Comment by roycaihw** [comment]
these files are generated by [swagger-codegen](https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources/python). The changes will be overwritten when we regenerate the client

you could send a PR to swagger-codegen to fix it upstream, and update the [patch](https://github.com/mbohlool/client-python/blob/master/scripts/rest_client_patch.diff) to carry it in the meantime

File: kubernetes/client/configuration.py [hunk 1]
```diff
@@ -106,6 +106,8 @@ def __init__(self):
         # Safe chars for path_param
         self.safe_chars_for_path_param = ''
 
+        self.retries = None
```
Comment (roycaihw) [inline kubernetes/client/configuration.py]: add documentation

**Comment by chowmean** [comment]
Created the PR in upstream. https://github.com/swagger-api/swagger-codegen/pull/9284

File: kubernetes/client/configuration.py (hunk 1 above)
Comment (chowmean) [reply kubernetes/client/configuration.py]: Added the doc.

**Comment by chowmean** [comment]
@roycaihw by update the patch to carry it in the meantime, did you mean this? 
git diff >> scripts/rest_client_retries_patch.diff
Need help here.

**Comment by wing328** [comment]
@chowmean do you mind filing a PR for https://github.com/OpenAPITools/openapi-generator as well?

I believe eventually we want to replace Swagger Codegen with OpenAPI Generator in this project: https://github.com/kubernetes-client/gen/issues/93

**Comment by chowmean** [comment]
@wing328 sure will do it..

**Comment by chowmean** [comment]
> @chowmean do you mind filing a PR for https://github.com/OpenAPITools/openapi-generator as well?
> 
> I believe eventually we want to replace Swagger Codegen with OpenAPI Generator in this project: [kubernetes-client/gen#93](https://github.com/kubernetes-client/gen/issues/93)

Created the PR. https://github.com/OpenAPITools/openapi-generator/pull/2460

**Comment by wing328** [comment]
UPDATE: OpenAPITools/openapi-generator#2460 has been merged into master. Thanks for the contribution by @chowmean

**Comment by roycaihw** [comment]
Thanks for working on this! The change will be included in this project when https://github.com/kubernetes-client/gen/pull/97 is in place

**Pull request closed without merging**